### PR TITLE
[MAINT] Add to codeowners

### PR DESCRIPTION
**Pull request by alexrockhill**
Add myself to codeowners

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -52,3 +52,15 @@
 # Non-tutorial documentation text and infrastructure
 /doc @larsoner @drammock
 /logo @drammock
+
+# Current-source density
```
Comment (larsoner) [inline .github/CODEOWNERS:56]: I tried to keep `mne/` toward the top and misc/other stuff toward the bottom, could you move these up?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -52,3 +52,15 @@
...
+/mne/preprocessing/_csd.py  @alexrockhill
+
+# Intracranial
+/mne/preprocessing/ieeg @alexrockhill
+*seeg*.py @alexrockhill
+*ecog*.py @alexrockhill
+
+# surface + transforms
+/mne/surface.py @alexrockhill
+/mne/transforms.py @alexrockhill
```
Comment (larsoner) [inline .github/CODEOWNERS:66]: Maybe also `_freesurfer.py`?

**Comment by larsoner** [comment]
Just pushed the changes and merged to avoid otherwise inevitable conflicts. I did add you to `_freesurfer.py` @alexrockhill let me know if you want me to revert that!

**Comment by alexrockhill** [comment]
Sounds good

**Pull request merged**